lib/src/values/values.dart

```diff
@@ -11,3 +13,61 @@ part 'logic_value.dart';
 part 'small_logic_value.dart';
 part 'big_logic_value.dart';
 part 'filled_logic_value.dart';
+
+/// Allows random generation of [LogicValue] for [BigInt] and [int].
+extension RandLogicValue on Random {
+  /// Generate non-negative random [BigInt] value that consists of
```

Comment:
I think more interesting than "non-negative" is "unsigned" (in all comments where you mention this)